tests/external_pyzeebe/test_client.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2010 New Relic, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import asyncio
+from pyzeebe import ZeebeClient, create_insecure_channel
+from pyzeebe.grpc_internals.zeebe_adapter import ZeebeAdapter
+from _mocks import (
+    dummy_create_process_instance,
+    dummy_create_process_instance_with_result,
+    dummy_deploy_resource,
+    dummy_publish_message,
+)
+
+from newrelic.api.background_task import background_task
+from testing_support.validators.validate_transaction_metrics import validate_transaction_metrics
+from testing_support.fixtures import validate_attributes
+
+
+client = ZeebeClient(create_insecure_channel())
+
+
+@validate_transaction_metrics(
+    "test_zeebe_client:run_process", rollup_metrics=[("ZeebeClient/run_process", 1)], background_task=True
+)
+@validate_attributes("agent", ["zeebe.client.bpmnProcessId"])
+def test_run_process(monkeypatch):
+    monkeypatch.setattr(ZeebeAdapter, "create_process_instance", dummy_create_process_instance)
+
+    @background_task(name="test_zeebe_client:run_process")
+    def _test():
+        response = asyncio.run(client.run_process("test_process"))
```

Comment:
This works to run this async function. Typically in the agent, we use `loop.run_until_complete`. Example: https://github.com/newrelic/newrelic-python-agent/blob/main/tests/external_aiobotocore/test_aiobotocore_s3.py#L110

Reply:
fixed in latest commit (switched to loop fixture)